connect/runtime/src/main/java/org/apache/kafka/connect/runtime/rest/entities/ConnectorInfo.java

```diff
@@ -71,7 +71,8 @@ public boolean equals(Object o) {
         ConnectorInfo that = (ConnectorInfo) o;
         return Objects.equals(name, that.name) &&
                 Objects.equals(config, that.config) &&
-                Objects.equals(tasks, that.tasks);
+                Objects.equals(tasks, that.tasks) &&
+                Objects.equals(type, that.type);
```

Comment:
We should update `hashCode` as well?

Reply:
@hachikuji You are right. Will update the PR asap